loops.js

```diff
@@ -5,6 +5,10 @@
 // Hoera!
 // Hoera!
 // ==========================================
+for (let i = 0; i <3; i++ ){
+    const hoera = 'hoera'
+    console.log(hoera);
+}
 
```

Comment:
`for (let i = 0; i < 3; i++) {
    console.log("Hoera")
}`
ik had hem zo gedaan. Werkt ook. Const hoeft niet er tusse. Hij print gewoon een string nu.